composer.json

```diff
@@ -33,17 +33,19 @@
         "http-interop/http-factory-guzzle": "Factory for guzzlehttp/guzzle"
     },
     "require-dev": {
-        "phpunit/phpunit": "^8.5",
-        "friendsofphp/php-cs-fixer": "^2.16",
+        "phpunit/phpunit": "^8.5 || ^9.0",
+        "bamarni/composer-bin-plugin": "^1.4.1",
```

Comment:
Is this package replacement mandatory to make this meilisearch-php working with PHP 8? If it's not, can you make these changes in another PR, please?

Reply:
Or even better, an issue so that we can discuss about the pros and cons of this integration before any work was done.

Reply:
I've reverted these changes.

I felt that moving php-cs-fixer out of this package's dev-dependenies using bamarni/composer-bin-plugin was the best move as discussed here (https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4702). However, I understand the concern and desire to have a discussion about this approach.

Reply:
Your change suggestion could be useful, and that's why I'm really interested in reading your arguments and discussing with the other external contributors maintaining this package 🙂 Opening a new issue is a perfect place for that!
Thanks for your involvement, it's really nice to have contributors like you trying to improve the package, not only fixing it!

Reply:
Love to hear that! I'll definitely open an issue. 👍